### PR TITLE
added new minmax vals for uncal_sntemp_input_output

### PR DESCRIPTION
**Pull request by msleckman**
Adding correct min max values. 

Note on this PR: `$seg_id_nat:3558` in this nc file was skewing values and it was removed before entering these data ranges.

### CODE REVIEW

File: in_text/text_03_driver.yml [hunk 1]
```diff
@@ -164,40 +164,40 @@ entities:
       attr-def: >-
          Area-weighted average air temperature for each segment from HRUs contributing flow to the segment
       attr-defs: https://doi.org/10.1002/joc.3413
-      data-min: NA
-      data-max: NA
+      data-min: 33.6
+      data-max: -26.15
```
Comment (jzwart) [inline in_text/text_03_driver.yml:168]: I think the min and max are switched here

File: in_text/text_03_driver.yml [hunk 2]
```diff
@@ -164,40 +164,40 @@ entities:
...
       data-units: degrees Celsius
     -
       attr-label: seg_tave_gw
       attr-def: >-
         Groundwater temperature.
       attr-defs: This data release
-      data-min: NA
-      data-max: NA
+      data-min: 28.32
+      data-max: -13.26
```
Comment (jzwart) [inline in_text/text_03_driver.yml:176]: switched min / max

File: in_text/text_03_driver.yml [hunk 3]
```diff
@@ -164,40 +164,40 @@ entities:
...
       data-units: degrees Celsius
     -
       attr-label: seg_tave_ss
       attr-def: >-
         Subsurface temperature.
       attr-defs: This data release
-      data-min: NA
-      data-max: NA
+      data-min: 31.36
+      data-max: -21.41
```
Comment (jzwart) [inline in_text/text_03_driver.yml:192]: switch min / max

File: in_text/text_03_driver.yml [hunk 4]
```diff
@@ -164,40 +164,40 @@ entities:
...
       data-units: degrees Celsius
     -
       attr-label: seg_tave_upstream
       attr-def: >-
         Temperature of streamflow entering each segment.
       attr-defs: This data release
-      data-min: NA
-      data-max: NA
+      data-min: 37.42
+      data-max: -98.9
```
Comment (jzwart) [inline in_text/text_03_driver.yml:200]: switch min / max

**Review by jzwart — CHANGES_REQUESTED** [review]
I just noted a couple locations where min / max are switched around, otherwise looks good!

File: in_text/text_03_driver.yml (hunk 1 above)
Comment (msleckman) [reply in_text/text_03_driver.yml:168]:
```suggestion
      data-min: -26.15
      data-max: 33.6
```

File: in_text/text_03_driver.yml (hunk 2 above)
Comment (msleckman) [reply in_text/text_03_driver.yml:176]:
```suggestion
      data-min: -13.26
      data-max: 28.32
```

File: in_text/text_03_driver.yml (hunk 3 above)
Comment (msleckman) [reply in_text/text_03_driver.yml:192]:
```suggestion
      data-min: -21.41
      data-max: 31.36
```

File: in_text/text_03_driver.yml (hunk 4 above)
Comment (msleckman) [reply in_text/text_03_driver.yml:200]:
```suggestion
      data-min: -98.9
      data-max: 37.42
```

**Comment by msleckman** [comment]
> I just noted a couple locations where min / max are switched around, otherwise looks good!

Knew this may be present, the ref code I used had max listed first.

Thanks will merge now.

**Pull request merged**